src/service/iot_config.proto

```diff
@@ -365,6 +365,53 @@ message admin_remove_key_req_v1 {
 
 message admin_key_res_v1 {}
 
+message gateway_info {
+  /// The asserted h3 (string) location of the gateway.
+  // Empty string if the gateway is not found
+  string location = 1;
+  bytes address = 2;
+  bytes owner = 3;
+  bool is_full_hotspot = 4;
+  /// the transmit gain value of the gateway in dbi x 10
+  // For example 1 dbi = 10, 15 dbi = 150
+  int32 gain = 5;
+  region region = 6;
+  blockchain_region_params_v1 region_params = 7;
+}
+
+enum gateway_error_reason {
+  reason_none = 0;
+  gateway_not_found = 1;
+}
+
+message gateway_error_res {
+  bytes address = 1;
+  gateway_error_reason error = 2;
+}
```

Comment:
we should be able to omit these; tonic can return a more standard http/grpc-level error response with the `Status` type, `not_found` being one of them. i would just make the response message type be the gateway_info struct, the timestamp, and we'll need a signature i would imagine since these will no longer be host-local the way the blockchain-node was running

Reply:
grpc protocol level error msgs are limited in the context they provide.  Perhaps they are sufficient for the purpose of the config service but if responses in any way need to be attestable  then application level error msgs may be required.  Something to keep in mind

Reply:
addressed here: e8f3a601a1ff0713430ee9e1b70406f3efeb39ff